execution_workflows/Roar/workflow/envs/Roar.yaml

```diff
@@ -0,0 +1,65 @@
+name: base
```

Comment:
Do we really need this environment  file? Because we would start in the APAeval env anyway, and this includes the ones here (just peeking, did not count). Also, Roar itself is not included, so maybe best to delete this and use the Docker image straightaway.